app/src/main/res/values/strings.xml

```diff
@@ -822,8 +830,21 @@
     <string name="onboarding_tracking_protection_header">Protect yourself</string>
     <!-- text for the tracking protection card description
     The first parameter is the name of the app (e.g. Firefox Preview) -->
-    <string name="onboarding_tracking_protection_description">%s blocks ad trackers that follow you around the web.
-    </string>
+    <string name="onboarding_tracking_protection_description1">%s helps stop websites from tracking you online.</string>
+    <!-- text for tracking protection radio button option for standard level of blocking -->
+    <string name="onboarding_tracking_protection_standard_button">Standard</string>
+    <!-- text for standard blocking option button description -->
+    <string name="onboarding_tracking_protection_standard_button_description">Blocks fewer trackers but allows pages to load normally</string>
+    <!-- text for tracking protection radio button option for strict level of blocking -->
+    <string name="onboarding_tracking_protection_strict_button">Strict (recommended)</string>
+    <!-- text for strict blocking option button description -->
+    <string name="onboarding_tracking_protection_strict_button_description">Blocks more trackers for better protection and performance, but may cause some sites to not work properly</string>
+    <!-- text for the toolbar position card header
+        In English this is an idiom for "choose a side as in an argument or fight"
+        but it is ok to make this more literally about "choosing a position in a physical space -->
+    <string name="onboarding_toolbar_position_header">Take a position</string>
+    <!-- text for the toolbar position card description -->
+    <string name="onboarding_toolbar_position_description">Try one-handed browsing with the bottom toolbar.</string>
```

Comment:
Isn't this : "Try one-handed browsing with the bottom toolbar or move it to the top. " ?